### ISSUE TITLE
Issues about EdmLib/ODataLib annotation Support

### ISSUE THREAD

**Issue by LaylaLiu** [OData/odata.net#37]
```
1. Support Annotation for Schema
2. Support defaultvalue of term
3. EdmLib should support 
    ○ dynamic conditional expression "Comparison and Logical Operators"
    ○ NavigationPropertyPath
    ○ PropertyPath
    ○ AnnotationPath
4.Add Terms for Capabilities/Measure/Core
5.EdmLib doesn't report error When external targeting annotation and embedded annotation use same termName and qualifier.
5. ODataLib doesn't support instanceannotation when query a primtive/collection/enum property 
6. ODataLib doesn't support write instanceannotation on expandedFeed.
7. ODataLib doesn't support metadata annotation of external targeting on Function or on properties and navigation properties of singletons or entities in a particular entity set.
<Annotations Target=""OperationTestService.FunctionBoundToCollectionOfEntity"" Qualifier=""Q1"">
     ...
</Annotations>
MySchema.MyEnumType/MyMember
MySchema.MyEntityContainer/MyEntitySet/MyProperty
MySchema.MyEntityContainer/MyEntitySet/MyNavigationProperty
MySchema.MyEntityContainer/MyEntitySet/MySchema.MyEntityType/MyProperty
MySchema.MyEntityContainer/MyEntitySet/MySchema.MyEntityType/MyNavProperty
MySchema.MyEntityContainer/MyEntitySet/MyComplexProperty/MyProperty
MySchema.MyEntityContainer/MyEntitySet/MyComplexProperty/MyNavigationProperty
MySchema.MyEntityContainer/MySingleton/MyComplexProperty/MyNavigationProperty
```

**Comment by MrTomWhite** [comment]
@laylaliu has there been any progress on this issue?

**Comment by Laylaliu** [comment]
@MrTomWhite These issue will not be fixed in ODataLib 6.10

**Comment by habbes** [comment]
Closing this stale issue. If this issue is still a concern in the latest version of the library, feel free to re-open or create a new issue.
